netket/experimental/driver/tdvp.py

```diff
@@ -153,9 +165,8 @@ def __init__(
         self.linear_solver = linear_solver
         self.linear_solver_restart = linear_solver_restart
 
-        self._w = self.state.parameters
         self._dw = None  # type: PyTree
-        self._last_qgt = None
+        self._last_qgt = self.qgt(self.state)
```

Comment:
A common pattern I use with other drivers is to construct and THEN set the initial parameters/reset the parameters of the state (as I know the driver is holding a reference to it).
I think this kind of approach does not work *at all* with TDVP driver, right?

Reply:
I'm not sure what you mean. This is only assigned here because JAX needs type-stability, so `None` didn't work. There is maybe a better initial value for this?

Reply:
But we are not hitting the driver itself, aren't we?
How is it possible that jax complains about this?